message ExecuteSectionsUplinkPayload {
  required bool post_buffer = 1 [default = false];
}